dashboard-app/src/api/index.ts

```diff
@@ -35,12 +35,19 @@ export const CbAdmins = {
 };
 
 export const CommunityBusinesses = {
+  configs: {
```

Comment:
What's the benefit of separating out the method and url?

Reply:
It's more a question of having a static declaration of the requests. 

If we only have functions that make the requests, then `useBatchRequest` needs to take `<T>(...a: any) => Promise<T>` as arguments, at which point it becomes less `useBatchRequest` and more `useBatchAsync` since there's no way of restricting those functions to only making API calls (except by manual inspection).

If we're gonna use hooks to do all API calls (eventually), then I imagine we can (also eventually) get rid of the function calls and just keep the static declarations.

Reply:
...unless we make the it accept `(...a: any) => Promise<AxiosResponse>`. But then there's still no guarantee that the function isn't doing arbitrary side effects.